ml_perf/flags/9/bootstrap.flags

```diff
@@ -7,6 +7,4 @@
 
 # Don't perform holdout for the first bootstrap round.
 --holdout_pct=0
-
-# Don't look for signal in what is known to be noise.
---num_readouts=50
+--num_readouts=250
```

Comment:
is there a reason `num_readouts` is larger than the 160 used for selfplay?

Reply:
no, this could be turned down again, esp if we're going to use normal distribution.